spark/src/main/scala/geotrellis/spark/io/json/package.scala

```diff
@@ -49,19 +50,20 @@ package object json {
     def write(metaData: RasterMetaData) = 
       JsObject(
         "cellType" -> metaData.cellType.toJson,
-        "extent" -> metaData.extent.toJson,
-        "crs" -> metaData.crs.toJson,
-        "tileLayout" -> metaData.tileLayout.toJson
+        "dataExtent" -> metaData.dataExtent.toJson,
+        "layoutExtent" -> metaData.layoutExtent.toJson,
+        "tileLayout" -> metaData.tileLayout.toJson,
```

Comment:
Probably be better to have `LayoutDefinition` have it's own JSON format, so that it matches the object more and then `LayoutDefinition` could have it's JSON used other places.